README.md

```diff
@@ -62,3 +62,19 @@ or start on the [landing page](http://localhost:8000/)
 14. README.md should contain a link to your personal docker hub repository win an app image
 15. README.md should contain instructions on how to access the application via a browser.
 16. Create PR with your changes and attach it for validation on a platform
+
+In order to start application you should execute the next steps:
+1. You must have Docker installed
+2. Start your Docker application
+3. Download the next docker images. Execute the next commands in terminal
+docker push yaaszp/todoapp:2.0.0
+docker push yaaszp/mysql-local:1.0.0
```

Comment:
These commands will not download docker images. You could remove them. Images will be automatically downloaded on step of executing docker run command.

Reply:
fixed